interfaces/Portalicious/src/app/components/form/form-sidebar.component.html

```diff
@@ -18,6 +19,7 @@
       pFocusTrap
       [pFocusTrapDisabled]="!visible()"
       [pAutoFocus]="visible()"
```

Comment:
This `pAutoFocus` doesn't seem to be working. It's not the biggest deal (I couldn't get it to work in confirmation popups either) but maybe we can remove it then.

Reply:
Yeah, I think the whole feature of Prime-Autofocus is not compatible with the requirement "the first form element should not show an error"...

But I'll check if it isn't a crucial part of my fix...

Reply:
In the end it turns out my fix is a "manual recreation" of that `pAutofocus`-directive...

So I wonder witch edge-case we're not covering here now... But it seems to work... ;)